src/sentry/api/endpoints/organization_events_facets_performance.py

```diff
@@ -75,6 +75,12 @@ def data_fn(offset, limit):
                 if not results:
                     return {"data": []}
 
+                for row in results["data"]:
+                    row["tags_key"] = tagstore.get_standardized_key(row["tags_key"])
+                    row["tags_value"] = tagstore.get_tag_value_label(
+                        row["tags_key"], row["tags_value"]
+                    )
```

Comment:
I think you wanna reverse these, `get_tag_value_label` depends on the key starting with `sentry:user` but `get_standardized_key` will strip out `sentry:`
```suggestion
                    row["tags_value"] = tagstore.get_tag_value_label(
                        row["tags_key"], row["tags_value"]
                    )
                    row["tags_key"] = tagstore.get_standardized_key(row["tags_key"])
```